doc/tutorial/read_mapper/index.md

```diff
@@ -0,0 +1,163 @@
+# Implementing your own read mapper with SeqAn3 {#tutorial_read_mapper}
+
+<b>Learning Objective:</b><br>
+In this tutorial you will learn how combine the taught components of previous tutorials to create your very first
+SeqAn3 application: a read mapper!
+
+\tutorial_head{High, 90 Minutes, All,}
+
+[TOC]
+
+<br><br>
+
+# Introduction
+
+Read mapping is a common task in bioinformatics and is often the first step of an in-depth analysis. Its aim is to
```

Comment:
```suggestion
Read mapping is a common task in bioinformatics and is often the first step of an in-depth analysis of [Next Generation Sequencing](https://de.wikipedia.org/wiki/DNA-Sequenzierung#Moderne_Ans%C3%A4tze) data. Its aim is to
```

Reply:
I'll add it. But I'll use the English wikipedia 🤣

Reply:
ops :see_no_evil: :laughing: